.github/workflows/main.yml

```
@@ -141,6 +145,11 @@ jobs:
         with:
           name: diff
           path: diff
+      # production dnslink is updated on release (during tag build)
+      - run: npx dnslink-dnsimple --domain dist.ipfs.io --link /ipfs/${{ steps.cid-reader.outputs.CID }}
```

Before merge, we need to add this, so we update dnslink only when master is built (and not  on PRs and tags)
```suggestion
      - run: npx dnslink-dnsimple --domain dist.ipfs.io --link /ipfs/${{ steps.cid-reader.outputs.CID }}
        if: github.ref == 'refs/heads/master'
```